src/app/derive/derive.component.html

```diff
@@ -0,0 +1,36 @@
+<app-banner></app-banner>
+
+<div class="container home-container" *ngIf="globalVars.inTab || globalVars.webview">
```

Comment:
I don't love that this code is duplicated. Can we just reuse the log-in component and add a `globalVars.derive` query param?